pandas/tests/io/test_excel.py

```diff
@@ -60,6 +59,20 @@ def setup_method(self, datapath):
         self.tsframe = _tsframe.copy()
         self.mixed_frame = _mixed_frame.copy()
 
+
+@td.skip_if_no('xlrd', '1.0.0')
+class ReadingTestsBase(SharedItems):
```

Comment:
It might not be entirely apparent from the diff but to clarify here I've moved `get_csv_refdf`, `get_excelfile` and `get_exceldf` out of the SharedItems class because they were only used by the ReaderBaseClass. 

What's left in SharedItems should be replaced by fixtures where needed in the next PR